runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -477,28 +477,26 @@ public String toString() {
 
 
     /**
-     * Given a SMR entry with an undo record, undo the update.
+     * Given a SMR entry with an undo entry, undo the update.
```

Comment:
Why change the name of record to entry?

Reply:
I renamed because this undoRecord is an SMREntry. In my following patch that unifies all three entry types, `SMREntry` will be rename into `SMRRecord`, then this will be renamed again.